### PR TITLE
6993: fix Editor bullet issue when using Quill 2.0.2

### LINKED ISSUE #6914: Editor: unordered list button creates ordered lists
### Describe the bug

In the rich text editor component, the unordered list button makes ordered lists

### Reproducer

_No response_

### PrimeReact version

10.7.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

Chrome 126, Firefox 128

### Steps to reproduce the behavior

1. Go to https://primereact.org/editor/
2. In either of the demo rich text editors, use the unordered list button, and see that it starts a numbered list:
image

### Expected behavior

The unordered list button should create an unordered list

### LINKED ISSUE #6993: Editor: Listing as bullet display as numbered
### Describe the bug

https://primereact.org/editor/
Listing items as bullet but display numbered.

image

### Reproducer

_No response_

### System Information

Chrome: v127 latest
    "primeflex": "^3.3.1",
    "primeicons": "^7.0.0",
    "primereact": "^10.8.2",

### Steps to reproduce the behavior

_No response_

### Expected behavior

Display bullet for Editor when hitting Unordered list

### PR DESCRIPTION
**Pull request by swotiendang**
### Defect Fixes
Fix #6993
Fix #6914

### Feature Requests
Current Quill editor in primereact 10.8.2 use css of Quill 1.3.7 which is obsoleted in newer version, this fix is to demonstrate how original Editor works in primereact.

Screenshot
image

### CODE REVIEW

**Comment by melloware** [comment]
actually @swotiendang they want you to copy quill Snow into that styles.css section so its CSS in JS and loaded as part of the component.  Can you copy the CSS into that block for 2.0X?

File: components/lib/editor/Editor.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { useMergeProps, useMountEffect, useUpdateEffect } from '../hooks/Hooks';
 import { DomHandler, classNames } from '../utils/Utils';
 import { EditorBase } from './EditorBase';
 import { useRef } from 'react';
+import 'quill/dist/quill.snow.css';
```
Comment (melloware) [inline components/lib/editor/Editor.js:8]: Remove this and copy the CSS into the block below

**Review by melloware — CHANGES_REQUESTED** [review]

File: components/lib/editor/Editor.js (hunk 1 above)
Comment (swotiendang) [reply components/lib/editor/Editor.js]: Updated

**Comment by swotiendang** [comment]
Do pay attention that this PR is quill snow original theme, I do not provide any modification for theme of primereact

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
I think PR has overrides already built into their themes on top of this SNOW theme.  So I think your change is a good one.

**Pull request merged**